From cbbb62ddda5c189c225f96bf6b599b3b3e8c8252 Mon Sep 17 00:00:00 2001
From: Mingli Yu <mingli.yu@windriver.com>
Date: Wed, 3 Aug 2022 16:44:29 +0800
Subject: [PATCH] version.c: don't print build flags

Don't print the build flags to avoid collecting the build environment info.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Mingli Yu <mingli.yu@windriver.com>
---
 src/main/version.c | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/src/main/version.c b/src/main/version.c
index f1f1e87810..3ffcbb25a0 100644
--- a/src/main/version.c
+++ b/src/main/version.c
@@ -589,19 +589,6 @@ void version_print(void)
 		DEBUG2("  unknown");
 #endif
 
-		DEBUG2("Compilation flags:");
-#ifdef BUILT_WITH_CPPFLAGS
-		DEBUG2("  cppflags : " BUILT_WITH_CPPFLAGS);
-#endif
-#ifdef BUILT_WITH_CFLAGS
-		DEBUG2("  cflags   : " BUILT_WITH_CFLAGS);
-#endif
-#ifdef BUILT_WITH_LDFLAGS
-		DEBUG2("  ldflags  : " BUILT_WITH_LDFLAGS);
-#endif
-#ifdef BUILT_WITH_LIBS
-		DEBUG2("  libs     : " BUILT_WITH_LIBS);
-#endif
 		DEBUG2("  ");
 	}
 	INFO("FreeRADIUS Version " RADIUSD_VERSION_STRING);
-- 
2.25.1

